pkg/controller/nodelifecycle/metrics.go

```diff
@@ -29,6 +29,7 @@ const (
 	zoneSizeKey             = "zone_size"
 	zoneNoUnhealthyNodesKey = "unhealthy_nodes_in_zone"
 	evictionsNumberKey      = "evictions_number"
+	evictionsTotalKey       = "evictions_total"
```

Comment:
can you inline this? it's only used once so it's confusing when i can't see it in directly in the metric

Reply:
thank your review! I don't the mean of inline , Can you explain or tell me what to do, thanks

Reply:
Wouldn't it make sense to keep it as is for consistency's sake? The other `*Key` constants are only used once as well.

Reply:
@logicalhan Does it need to be modified here?

Reply:
For consistency purposes, I think it is fine to keep it there, although I agree with @logicalhan that it is confusing to have it defined there.

/lgtm

Reply:
Thanks, Done